feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go

```diff
@@ -328,7 +328,7 @@ func (a *testArgs) createFlow(name string, dst *attrs.Attributes) *ondatra.Flow
 }
 
 // validateAftTelmetry verifies aft telemetry entries.
-func (a *testArgs) validateAftTelemetry(t *testing.T, vrfName string, prefix string, ipaddress string) {
+func (a *testArgs) validateAftTelemetry(t *testing.T, vrfName string, prefix string, ipAddress, resolvedNhIpAddress string) {
```

Comment:
Can write the parameters a bit more compactly:
(t *testing.T, vrfName, prefix, ipAddress, resolvedNhIpAddress string)

Reply:
Updated the parameters.